<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchingScene</class>
 <widget class="QDialog" name="SearchingScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1978</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_image3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>1071</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-Final_Project-Desktop_Qt_5_12_12_MinGW_64_bit-Debug/SearchQuestion.PNG</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>1780</x>
     <y>20</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_searchName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>901</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>710</y>
     <width>911</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
